semver.js

```diff
@@ -421,6 +421,14 @@ SemVer.prototype.inc = function (release, identifier) {
       if (this.prerelease.length === 0) { this.patch++ }
       this.prerelease = []
       break
+    // This is for conversion from a '>' to '>='.
+    case 'gte':
+      if (this.prerelease.length === 0) {
+        this.patch++
+      } else {
+        this.prerelease.push(0)
+      }
+      break
```

Comment:
Could you change this patch so it doesn't add a subcommand like this to `semver.inc()`? This change makes it part of the external API.

Reply:
Thanks for your review.  I've made the requested change.